<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1632</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(179, 200, 207);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>641</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(190, 215, 220);</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>601</width>
       <height>591</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>60</y>
      <width>921</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(241, 238, 220);</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>581</width>
       <height>611</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>10</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(178, 178, 35);
border-radius: 2px;
font: 8pt &quot;8514oem&quot;;</string>
     </property>
     <property name="text">
      <string>ANGRY</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>40</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>90</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(178, 178, 35);
border-radius: 2px;
font: 8pt &quot;8514oem&quot;;</string>
     </property>
     <property name="text">
      <string>DISGUST</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_2">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>120</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>180</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(178, 178, 35);
border-radius: 2px;
font: 8pt &quot;8514oem&quot;;</string>
     </property>
     <property name="text">
      <string>FEARFUL</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_3">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>210</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>270</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(178, 178, 35);
border-radius: 2px;
font: 8pt &quot;8514oem&quot;;</string>
     </property>
     <property name="text">
      <string>HAPPY</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_4">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>300</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>370</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(178, 178, 35);
border-radius: 2px;
font: 8pt &quot;8514oem&quot;;</string>
     </property>
     <property name="text">
      <string>NEUTRAL</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_5">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>400</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_6">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>490</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>460</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(178, 178, 35);
border-radius: 2px;
font: 8pt &quot;8514oem&quot;;</string>
     </property>
     <property name="text">
      <string>SAD</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_7">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>570</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>540</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(178, 178, 35);
border-radius: 2px;
font: 8pt &quot;8514oem&quot;;</string>
     </property>
     <property name="text">
      <string>SURPRISED</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>19</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;8514oem&quot;;
padding-left: 2px</string>
    </property>
    <property name="text">
     <string>WEBCAM</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;8514oem&quot;;
padding-left: 2px</string>
    </property>
    <property name="text">
     <string>ANALYSIS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 44, 118);
border-radius: 2px;
font: italic 8pt &quot;8514oem&quot;;</string>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(184, 184, 184);
border-radius: 2px;
font: italic 8pt &quot;8514oem&quot;;</string>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 44, 118);
border-radius: 2px;
font: italic 8pt &quot;8514oem&quot;;</string>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;8514oem&quot;;
padding-left: 2px</string>
    </property>
    <property name="text">
     <string>PREDICT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(184, 184, 184);
border-radius: 2px;
font: italic 8pt &quot;8514oem&quot;;</string>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>700</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;8514oem&quot;;
padding-left: 2px</string>
    </property>
    <property name="text">
     <string>BY: EDIMAR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1632</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
